### PR TITLE
format fonts and colors - WiFi Wizard

### PR DESCRIPTION
**Pull request by jankrzempek**
https://trello.com/c/MQepK0pv/431-config-wizard-fixed-wifi-text-updates

### CODE REVIEW

File: AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsView.swift [hunk 1]
```diff
@@ -24,7 +24,7 @@ struct CreateSessionDetailsView: View {
                             sessionNameField
                             sessionTagsField
                         }
-                        if sessionContext.sessionType == SessionType.fixed { fixedSessionDetails }
+                        if sessionContext.sessionType == SessionType.mobile { fixedSessionDetails }
```
Comment (smialko) [inline AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsView.swift:27]: I think you added this for testing and forgot to change it back :D 
Please, change it - it's not working for fixed session right now 🙃

File: AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift [hunk 2]
```diff
@@ -35,7 +35,7 @@ class CreateSessionDetailsViewModel: ObservableObject {
         sessionContext.sessionName = sessionName
         sessionContext.sessionTags = sessionTags
         
-        guard sessionContext.sessionType == .fixed else {
+        guard sessionContext.sessionType == .mobile else {
```
Comment (smialko) [inline AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift:38]: please, change bakc to `.fixed`🐝

**Review by smialko — CHANGES_REQUESTED** [review]
:D there are some leftovers from testing, please, take a look

**Comment by smialko** [comment]
And the rest looks ok so you can merge after those fixes :D

File: AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsView.swift (hunk 1 above)
Comment (jankrzempek) [reply AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsView.swift:27]: yees. oh my :/

File: AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift (hunk 2 above)
Comment (jankrzempek) [reply AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift:38]: Thank U!

**Pull request merged**